package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "prod": "NODE_ENV=production webpack --config ./webpack/webpack.build.config.js && electron --noDevServer .",
     "dev": "webpack-dev-server --hot --host 0.0.0.0 --config=./webpack/webpack.dev.config.js",
+    "validate:lint": "tslint -p tsconfig.json",
```

Comment:
It would be good to add a tsc validation too: `"validate:tsc": "tsc -p tsconfig.json --noEmit --pretty"`.

Reply:
Fixed